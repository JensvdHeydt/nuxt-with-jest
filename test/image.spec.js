import Vue from 'vue'
import Image from '../components/image'

describe('Image Component', () => {
  let $mounted

  beforeEach(() => {
    $mounted = new Vue(Image).$mount()
  })

  test('snapshot', () => {
    let $html = $mounted.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
