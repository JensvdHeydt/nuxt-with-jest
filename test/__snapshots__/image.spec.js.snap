// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Component snapshot 1`] = `"<img alt=\\"image\\">"`;
